### DISCUSSION TITLE
Is it possible to have color filled in the area covered in the slider?

### DISCUSSION THREAD

**Discussion by TechieGeeke** [rdbende/Azure-ttk-theme#36] [Question] [answered]
Hello @rdbende ,
May I know if there is a way to change the covered area color in tkinter slider? If yes could you suggest me a way to do this. I have a great project coming soon. Glad if you would help me in this faster.

So what I am thinking of is, like in the image below:
![417-4179159_example-of-volume-slider-door-handle-hd-png](https://user-images.githubusercontent.com/95920517/171013097-48edc47c-180f-4403-bff0-46c48f30c778.png)

See that black color getting filled in the places that are covered??
More specifically, like how the progress bar works...

**Comment by rdbende** [comment, marked as answer]
No, unfortunately I don't know any way to make this. Maybe it's possible by combining a `Scale` and a `Progressbar` widgets, but I have no idea how to do it.

**Reply by TechieGeeke** [reply]
Oh its ok. I have got some ideas to work this out. But I'm still uncertain if this is practically possible. But with hope, there might be some way. I'll let you know if I could make some good news.
